### PR TITLE
ibmcloud | doc: Update ibmcloud doc to use operator approach

### PR DESCRIPTION
**Pull request by stevenhorsman**
- Move the old terraform documentation into the terraform directory
- Add new documentation for the operator based approach

### CODE REVIEW

File: ibmcloud/README.md [hunk 1]
```diff
@@ -1,603 +1,242 @@
...
+  ```
+  kubectl delete -f ibmcloud/demo/nginx.yaml
+  ```
+
+## Uninstall and clean up
+
+There are two options for cleaning up the environment once testing has finished, or if you want to re-install from a
+clean state:
+- If using a self-managed cluster you can delete the whole cluster following the
+[Delete the cluster documentation](./ibmcloud/cluster#delete-the-cluster) and then start again.
```
Comment (Jordan9500) [inline ibmcloud/README.md:212]: think this needs to be `./cluster#delete-the-cluster`

**Review by Jordan9500 — CHANGES_REQUESTED** [review]
LGTM Other than a few minor changes

**Comment by stevenhorsman** [comment]
@Jordan9500 - thanks so much for the review. I hope I've address your comment adequately, but please feel free to give further feedback. For convenience I've push the changes as a separate commit, but will squash it before merging.

**Review by Jordan9500 — APPROVED** [review]
LGTM! Thanks!

**Review by jtumber-ibm — APPROVED** [review]

**Pull request merged**